code/modules/examine/descriptions/weapons.dm

```diff
@@ -0,0 +1,81 @@
+/*
+	Note: This file is meant for actual weapons (guns, swords, etc), and not the stupid 'every obj is a weapon, except when it's not' thing.
+*/
+
+//******
+//*Guns*
+//******
+
+//This contains a lot of copypasta but I'm told it's better then a lot of New()s appending the var.
+/obj/item/weapon/gun
+	description_info = "This is a gun.  To fire the weapon, have your gun mode set to 'fire', \
+	then click where you want to fire."
+
+/obj/item/weapon/gun/energy
+	description_info = "This is an energy weapon.  To fire the weapon, have your gun mode set to 'fire', \
+	then click where you want to fire.  Most energy weapons can fire through windows harmlessly.  To recharge this weapon, use a weapon recharger."
+
+/obj/item/weapon/gun/energy/crossbow
+	description_info = "This is an energy weapon.  To fire the weapon, have your gun mode set to 'fire', \
+	then click where you want to fire."
+	description_antag = "This is a stealthy weapon which fires poisoned bolts at your target.  When it hits someone, they will suffer a stun effect, in \
+	addition to toxins.  The energy crossbow recharges itself slowly, and can be concealed in your pocket or bag."
+
+/obj/item/weapon/gun/energy/gun
+	description_info = "This is an energy weapon.  To fire the weapon, have your gun mode set to 'fire', \
+	then click where you want to fire.  Most energy weapons can fire through windows harmlessly.  To switch between stun and lethal, click the weapon \
+	in your hand.  To recharge this weapon, use a weapon recharger."
+
+/obj/item/weapon/gun/energy/gun/taser
```

Comment:
Not accessible to players; might want to make this the hybrid taser and ensure that the description mentions it has a stun and a disable mode.